packages/dds/sequence/package.json

```diff
@@ -193,6 +193,7 @@
 	},
 	"typeValidation": {
 		"broken": {},
+		"disabled": true,
```

Comment:
Why full stop disable? Isn't this exactly the case where we want to list the incompatibilities? Is another option to change entrypoint from "legacy" to "public"?

Reply:
i don't want to deal with merge conflict right now. once we are closer to the merge point i'll remove it and deal with them